examples/resources/rafay_project/resource.tf

```diff
@@ -17,6 +17,7 @@ resource "rafay_project" "tfdemoproject2" {
   }
   spec {
     default = false
+    sync_excluded_namespaces = ["one", "two"]
```

Comment:
instead of `one,two` can rename to `ns1,ns2,ns3` something like this..

Reply:
can you update https://github.com/RafaySystems/terraform-provider-rafay/blob/master/docs/resources/project.md doc also..?

Reply:
okay, will make that change.